src/components/general/BookmarksManager/BookmarkSideSheet/SideSheetManager/Bookmark/index.tsx

```diff
@@ -110,6 +104,27 @@ export const Bookmark = ({
                 hideCancelAction: true,
                 body: <ShareBody />,
             });
+            try {
+                store.updateBookmark(bookmark.id, {
```

Comment:
havent looked in store, but does this function throw error?

Reply:
In the epic for updateBookmark there is `catchError((error) => of(failure({ action, error }))),` - so maybe no need for the try catch since it's already handled in the epic?